src/main/fc/fc_core.c

```diff
@@ -884,6 +903,11 @@ static NOINLINE void subTaskMainSubprocesses(timeUs_t currentTimeUs)
         updateMagHold();
     }
 #endif
+
+#ifdef USE_GPS_RESCUE
+    updateRcCommands();
```

Comment:
I don't think that this is needed here - `updateRcCommands()` is called right before `updateArmingStatus()`.

Reply:
If i remember correctly, it needs to be called after updateRescueState() runs.  I'll look into whether this can be optimized or removed.